Dockerfile.k8s

```diff
@@ -0,0 +1,50 @@
+# Build Iter8
```

Comment:
I think we need only one Dockerfile. 

Perhaps `mv Dockerfile.k8s Dockerfile`? 

Not sure what the difference is but the move will reveal the changes.

Reply:
only real difference was build target. replaced original.